chingu-voyage4/Geckos-Team-32
# Profile

## Purpose
* Beginning of Profile write

## Approach/Fixes
* First half of profile page done

## Testing
* Download branch
* Open profile
* Check username editing and overall layout
* Beginnings of avatar selection, no js written yet
